sample/src/portfolio/components/reagent.cljs

```diff
@@ -4,7 +4,8 @@
   ;; For react versions 18+ use the following:
   ;; This is due to the new API https://www.metosin.fi/blog/reagent-towards-react-18/
   (:require [portfolio.components.reagent.component :as rc]
-            [portfolio.reagent-18 :refer-macros [defscene]]))
+            [portfolio.theme :as theme]
+            [portfolio.reagent :refer-macros [defscene]]))
 
```

Comment:
Why did you change the React version in use here?

Reply:
it's maybe lsp related mistake. i will revert it.